src/main/java/frc5124/robot2020/subsystems/Hanger.java

```diff
@@ -19,15 +19,17 @@
 
 public class Hanger implements Subsystem {
   private CANSparkMax hangerMotor;
-  private Solenoid brake;
+  private Solenoid noBrake;
```

Comment:
How about naming it "brakeReleaser" or "unbraker"? This commit change looks like manufacturing decided to take the brake off the robot so we the programmers programmed it anyways or smth.